aeolis/constants.py

```diff
@@ -31,14 +31,17 @@
         'uw',                               # [m/s] Wind velocity
```

Comment:
it appears to me that these changes cannot do any harm, i dont undestand the 'required parameters' entry.